cmd/nodelink-controller/main.go

```diff
@@ -61,15 +61,15 @@ const (
 	controllerName = "nodelink"
 
 	// machineAnnotationKey is the annotation storing a link between a node and it's machine. Should match upstream cluster-api machine controller. (node.go)
-	machineAnnotationKey = "cluster.k8s.io/machine"
+	machineAnnotationKey = "machine.openshift.io/machine"
```

Comment:
Time to use https://github.com/openshift/cluster-api/blob/release-4.0/pkg/controller/node/node.go#L35 instead.